app/views/exchanges/hbx_profiles/_view_hbx_enrollments.html.erb

```diff
@@ -89,12 +87,12 @@
   </div>
   <script type="text/javascript">
     var ReinstateModal = function () {
-      var failure = document.querySelector('tr[data-cant-reinstate] input[type=radio]:checked');
+      var row = document.querySelector('.reinstate-enrollment-row input[type=radio]:checked');
```

Comment:
This line actually leads to every row being marked a success as instead of selecting the row with the data-attribute attached, you are selecting the input. In order to resolve you can either adjust this line to then get the parent row (`var row = document.querySelector('.reinstate-enrollment-row input[type=radio]:checked').closest(".reinstate-enrollment-row");`) or to move the failure reason to the input